package-lock.json

```diff
@@ -3436,8 +3436,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
+          "bundled": true
```

Comment:
I'm guessing these package-lock.json changes were not deliberate, I still don't fully understand why these changes pop up from time to time but I'm thinking it'd be best to leave them out of this PR to minimize the number of times this file changes unnecessarily.